### PR TITLE
Add data_converter features in Java (#81)

### PR DESCRIPTION
**Pull request by antlai-temporal**
## What was changed
Adding Java data converter features.

1. Closes 

Progress to closing #81

### CODE REVIEW

File: features/data_converter/binary/feature.java [hunk 1]
```diff
@@ -1,34 +1,30 @@
...
 import java.nio.file.Files;
 import java.nio.file.Paths;
 
-import com.google.protobuf.util.JsonFormat;
-
 @WorkflowInterface
 public interface feature extends Feature {
-    static byte[] deadbeef = new byte[]{(byte)0xde, (byte)0xad, (byte)0xbe, (byte)0xef};
-    
+  static byte[] deadbeef = new byte[] {(byte) 0xde, (byte) 0xad, (byte) 0xbe, (byte) 0xef};
```
Comment (Quinn-With-Two-Ns) [inline features/data_converter/binary/feature.java:18]: nit: `static` is redundant here, also this variable is a constant so convention is all caps with underscore to separate words

File: features/data_converter/binary/feature.java [hunk 2]
```diff
@@ -1,34 +1,30 @@
...
+
   @WorkflowMethod
   public byte[] workflow();
```
Comment (Quinn-With-Two-Ns) [inline features/data_converter/binary/feature.java:21]: bit: `public` is redundant here

File: features/data_converter/binary/feature.java (hunk 1 above)
Comment (Quinn-With-Two-Ns) [reply features/data_converter/binary/feature.java:18]: I know you didn't write this code, but you did the same thing in some of the other features. Not a blocker

**Review by Quinn-With-Two-Ns — APPROVED** [review]

File: features/data_converter/binary/feature.java (hunk 1 above)
Comment (antlai-temporal) [reply features/data_converter/binary/feature.java]: Thanks, it makes sense, done...

File: features/data_converter/binary/feature.java (hunk 2 above)
Comment (antlai-temporal) [reply features/data_converter/binary/feature.java]: Yes, it is an interface... Done

**Pull request merged**